tests/shared/crclient/client.go

```diff
@@ -122,6 +124,15 @@ func (f Client) listPodList(list *corev1.PodList) error {
 	return nil
 }
 
+func (f Client) listNodeList(list *corev1.NodeList) error {
+	for k, v := range f.Objects {
+		if k.Kind == "Node" {
+			list.Items = append(list.Items, *v.(*corev1.Node))
```

Comment:
Is there a chance for this type conversion to panic? Maybe something like `node, ok := *v.(*corev1.Node); ok` is safer.

Reply:
thanks Aaron. this is fixed in https://github.com/dell/csm-operator/pull/811/commits/21d5b069a5b340b78a52ae2ca24133dcab88f84f